karpenter-operator/controllers/karpenter/assets/adjust-cel.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
can we include this in make in such a way that makes fail verify.
E.g. karpenter-api: and let it be a dep of api:

Reply:
```
./karpenter-operator/controllers/karpenter/assets/adjust-cel.sh: line 6: yq: command not found
```
verify is failing because `yq` doesn't exsit in CI